lib/stacked_card_carousel.dart

```diff
@@ -114,10 +126,11 @@ class _StackedCardCarouselState extends State<StackedCardCarousel> {
               opacity = factor < 0.0 ? 0.0 : pow(factor, 1.5).toDouble();
               scale = factor < 0.0 ? 0.0 : pow(factor, 0.1).toDouble();
             }
-            return Positioned.fill(
-              top: -position,
+            return Positioned(
```

Comment:
❗️ This part of the code should be reconsidered since it breaks also current behavior. Please check the attached gif with old and new behavior. Look at the top part of stacked cards.

Old:
![old](https://user-images.githubusercontent.com/5623366/135760238-9359396c-2fd5-47a4-88a1-72c7eb5a3fa9.gif)

New:
![new](https://user-images.githubusercontent.com/5623366/135760255-08d6d657-ec35-4b9d-bbfe-1f531936cacb.gif)